packages/services/f-statistics/src/index.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable camelcase */
+import defaultOptions from './defaultOptions';
+
+export default class StatisticsClient {
+    constructor (justLogInstance, options = {}) {
```

Comment:
Just a niggly bit `justLogInstance` could be named more generic...

Reply:
At the moment, it has to be a provided just log instance; we might in future make it more variable; but at the moment its just a dumb wrapper that only works with that provider.